src/odemis/gui/model/__init__.py

```diff
@@ -1202,16 +1244,31 @@ def moveFocusRel(self, shift):
         """
         # FIXME: "stop all axes" should also clear the queue
 
+        # If streams have a z-level, we calculate the shift differently.
+        
+        if hasattr(self, "zpos"):
+
+            # Multiplier found by testing based on the range of zpos
```

Comment:
So, you can scan the whole range by moving the mouse by 300 px? Right? Maybe indicate that more clearly in the comment.
Also, can you write `300` instead of `3e2`. It's a little more readable ;-)